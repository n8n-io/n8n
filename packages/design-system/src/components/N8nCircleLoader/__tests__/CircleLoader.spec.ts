import { render } from '@testing-library/vue';
import N8NCircleLoader from '../CircleLoader.vue';

describe('N8NCircleLoader', () => {
	it('should render correctly', () => {
		const wrapper = render(N8NCircleLoader, {
			props: {
				radius: 20,
				progress: 42,
				strokeWidth: 10,
			},
		});
		expect(wrapper.html()).toMatchSnapshot();
	});
});
