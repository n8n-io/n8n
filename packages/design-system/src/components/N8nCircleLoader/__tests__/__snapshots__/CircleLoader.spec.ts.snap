// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`N8NCircleLoader > should render correctly 1`] = `
"<div class=\\"progress-circle\\"><svg class=\\"progress-ring\\" width=\\"60\\" height=\\"60\\">
    <circle class=\\"progressRingCircle\\" stroke-width=\\"10\\" stroke=\\"#DCDFE6\\" fill=\\"transparent\\" r=\\"20\\" cx=\\"30\\" cy=\\"30\\"></circle>
    <circle class=\\"progressRingCircle\\" stroke=\\"#5C4EC2\\" stroke-width=\\"10\\" fill=\\"transparent\\" r=\\"20\\" cx=\\"30\\" cy=\\"30\\" style=\\"stroke-dasharray: 125.66370614359172; stroke-dashoffset: 72.8849495632832;\\"></circle>
  </svg></div>"
`;
